source/agent/video/videoMixer/videoMixer_sw/binding.sw.gyp

```diff
@@ -53,12 +53,6 @@
             'sources': [
                 '../../../../core/owt_base/SVTHEVCEncoder.cpp',
             ],
```

Comment:
Why remove the libs link?

Reply:
i installed them to build/libdeps/build as other dependencies

Reply:
Suggest to keep source code include path unchanged, and only modify .gyp

Reply:
link updated